troll-o-matic.py

```diff
@@ -157,7 +163,7 @@ def do_review(self, review_type, change, fixes_ref, msg, notify, vote):
     final_msg += self.STRING_FOOTER
 
     self.inc_stat(review_type)
-    if not self.args.dry_run:
+    if not self.args.dry_run and vote:
```

Comment:
Hmm, this isn't what we want. There are cases where we want to clear a previous vote (such as clearing a -1 when missing fields have been added, or when prefix has been fixed), this will also make my terminal very sad :)

Reply:
Fixed by setting final_msg="" in this case.

I did some experiments with https://chromium-review.googlesource.com/c/chromiumos/third_party/kernel/+/1528250 and this looks ok. Votes are cleared, and this does not add any blank message upon review/repeated review.